packages/react/src/components/Callout/Callout.types.ts

```diff
@@ -33,7 +33,7 @@ export interface ICalloutProps extends React.HTMLAttributes<HTMLDivElement>, Rea
   directionalHintForRTL?: DirectionalHint;
 
   /**
-   * The gap between the Callout and the target
+   * The gap between the Callout and the target, specified as number of pixels
```

Comment:
Should this comment also address the question about negative values raised in the issue?

Reply:
I think that one is easier to be checked if you play with the component. It's really easy to pass a negative value and just see that it does work and does what you expect.